muxpi/dut.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+rm -f $HOME/serial-output
+echo "Setting up DUT" 
+
+#Clear muxpi display and print 'DUT'
+
+stm_armv7 -clr #clear display
+stm_armv7 -print "DUT"
+
+#Make the SD card adapter emulate a real SD-card. Note that the SD card wont be visible on muxpi when its on DUT mode.
+
+stm_armv7 -dut
+echo "RPI booting..."
+sleep 30s
+echo "Starting UART screens..."
+
+stty -F /dev/ttyACM0 115200 cs8 -cstopb -parenb
+cd $HOME
```

Comment:
You can remove this `cd $HOME` and just append a $HOME/ to serial-output below.

Or better yet, add a variable at the top of the file and use it in the `rm` above and when piping output below.

Reply:
When using $HOME/something it gives a warning about a slash '/'. Thats why i decided to cd there since the output is more clean.